content/information-distribution/source.md

```diff
@@ -75,6 +76,33 @@ In such an event, client implementations **should** consider articles posted
 prior to the key being declared as compromised as probably not having been
 tampered with, with no way to be 100% sure about it.
 
+## Change of Matrix user
+
+A source or subsource might have to change the Matrix user it uses to publish
+information at some points in its life time. This can happen for many reason:
+the homeserver it was registered on went down, or got compromised, or got
+isolated from the rest of the federation...
+
+In such an event, the source or subsource **must** publish a new registration
+event from its new Matrix user with the `prev_id` key having its previous user's
+ID as its value.
```

Comment:
I think you should add a matrix event ID alongside `prev_id`, in order to invalidate events from the compromised source after a given date (or event in this case), while keep using the older ones.

Reply:
Maybe also add a reason, to let the source explain why they changed their account